src/olupis/content/NyfalisBlocks.java

```diff
@@ -385,10 +385,10 @@ public static void LoadWorldTiles(){
             lightColor = Color.valueOf("D54B3B").a(0.38f);
         }};
 
-        mossyWater = new Floor("mossy-water"){{
+        algaeWater = new Floor("algaewater-shallow"){{
```

Comment:
This would break existing maps & saves
Remaining the name within `"` changes it to a technically different block

Reply:
reverted important changes